exercises/practice/hello-world/.docs/instructions.md

```diff
@@ -0,0 +1,16 @@
+# Instructions
+
+The classical introductory exercise.
+Just say "Hello, World!".
+
+["Hello, World!"][hello-world] is the traditional first program for beginning programming in a new language or environment.
+
+The objectives are simple:
+
+- Modify the provided code so that it produces the string "Hello, World!".
+- Run the test suite and make sure that it succeeds.
+- Submit your solution and check it at the website.
+
+If everything goes well, you will be ready to fetch your first real exercise.
```

Comment:
This is a real exercise, in that it is exercising and orienting to the track.  Should we not diminish that importance, and say something to the effect of "continue on" or something?

Reply:
This is not necessarily meant to change here, as I am pretty sure it is coming from upstream, and is only something I noted.

Reply:
This is a ready-made script, I didn't write it, but we can change it.

Reply:
Ah, script as in "prose".  Yeah, I presume it is coming from upstream.  Just be aware in case you are mentoring and someone were to mention it.  Not a stopper at all, and has survived this long, so we should not let it stop us, and even does not require a change here, and may not require a change at all anywhere (though I am likely to forget that I saw it and notice it again months down the road).

Reply:
So yes, there is no harm for now, what we are talking about is optimization and even those who contribute over time can fix it. It doesn't take me a long time to fix it, it's just that if I get too caught up in the details I get distracted from moving forward.

Reply:
That is fine, it is coming from upstream, here as a note, and no need to fix it here at all, unless one would want to.  Eventually, if fixed upstream, it will trickle down to here, potentially, to be better in both places.